src/app/modules/estadisticas/components/citas/filtros.component.ts

```diff
@@ -101,12 +105,20 @@ export class FiltrosComponent implements AfterViewInit, OnChanges {
 
     constructor(
         private plex: Plex,
+        public auth: Auth,
         public servicioProfesional: ProfesionalService,
         public servicioPrestacion: TipoPrestacionService) {
     }
 
     ngAfterViewInit() {
-        this.onChange();
+        if (!this.verProfesionales) {
+            let query = {
```

Comment:
Los datos del profesional ya estan en:

`this.auth.profesional`

Reply:
biem!